<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddComponents</class>
 <widget class="QDialog" name="AddComponents">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_typeComponents">
         <property name="text">
          <string>Тип комплектующего</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_ComponentName">
         <property name="text">
          <string>Копмлектующее</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_description">
         <property name="text">
          <string>Описание</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QComboBox" name="comboBox_typeComponent"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_componentName"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_description"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="button_addComponent">
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
